File: tests/inputs/15_test_everything.c
int ehPar(int n) {
    if (n % 2 == 0) {
        return 1;
    } else {
        return 0;
    }
}

int somaImpares(int limite) {
    int soma = 0;
    for (int i = 1; i <= limite; i++) {
        if (ehPar(i)) {
            continue;
        }
        soma = soma + i;
    }
    return soma;
}

int main() {
    int resultado = somaImpares(10);
    int x = 1;
    int y = 0;
    int valido = (resultado > 0) && !(x == y);
    return 0;
}

File: tests/expected_outputs/15_test_everything.py
def ehPar(n):
    if n % 2 == 0:
        return 1
    else:
        return 0

def somaImpares(limite):
    soma = 0
    i = 1
    while i <= limite:
        if ehPar(i):
            i += 1
            continue
        soma = soma + i
        i += 1
    return soma

def main():
    resultado = somaImpares(10)
    x = 1
    y = 0
    valido = (resultado > 0) and not (x == y)
    return 0